src/gui_common/tooltip/ToolTipHelper.cs

```diff
@@ -18,9 +20,14 @@ public static class ToolTipHelper
     /// <summary>
     ///   Instantiates a default tooltip scene
```

Comment:
This comment is incorrect now, should say this returns a cached tooltip instance or creates a new one if the cache is empty.